components/in_memory_engine/src/region_manager.rs

```diff
@@ -109,7 +109,16 @@ impl SnapshotList {
     }
 }
 
-#[derive(Debug)]
+/// Estimates the smoothed coprocessor request rate over the last hour using a
+/// simple moving average.
+pub(crate) type CopRequestsSMA = Smoother<f64, COP_REQUEST_SMA_RECORD_COUNT, ONE_HOUR_IN_SECS, 0>;
```

Comment:
```suggestion
pub(crate) type CopRequestsSma = Smoother<f64, COP_REQUEST_SMA_RECORD_COUNT, ONE_HOUR_IN_SECS, 0>;
```
Follow the rust naming convension